guides/release/state-management/tracked-properties.md

```diff
@@ -255,7 +255,7 @@ export default class ApplicationController extends Controller {
 ```js {data-filename=app/templates/application.hbs}
 {{this.model.fullName}}
 
-<button {{action this.updateName 'Krati' 'Ahuja'}}>
+<button {{on (fn this.updateName 'Krati' 'Ahuja')}}>
```

Comment:
`<button {{on "click" (fn this.updateName 'Krati' 'Ahuja')}}>`

Reply:
gasp! thanks!

Reply:
fixed!